news/_posts/2024-02-24-open-door.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: news
+title: "Lab members participate in ASU Open Door 2024"
+author: "Navish Wadhwa"
+author_handle: "navish"
+# image: /assets/images/news/microscope.jpg
+category: news
+tags: [people]
+---
```

Comment:
Are we planning to include images of the agar art?

Reply:
Yes, I forgot about this file that I had made. I will make changes and update.